### DISCUSSION TITLE
Conditional Fields

### DISCUSSION THREAD

**Discussion by shayanikeghbali** [mdelobelle/metadatamenu#722] [Q&A]
Hello,

Is it possible to use conditions to show different fields?
for example:
if I change the values in the "Cycle" field, I want to show other fields:

if Cycle field = A
  I want to have fields a-1, a-2, a-3
if Cycle field = B
  I want to have fields b-1, b-2, b-3
  
Is it possible to have this kind of field options?

BTW, I don't know anything about coding. I saw "Query-based FileClass settings" in the settings, but don't know if it's useful for what I want

**Comment by LynetteCullens** [comment]
You do this by adding sub-fileClasses to an already existing fileClass. The Cycle field will be the determining factor of the sub-fileClass. All sub classes will have the properties of the parent class, but the parent class does not have the same properties of the child class.
